spec/models/item_spec.rb

```diff
@@ -3,5 +3,11 @@
 RSpec.describe Item, type: :model do
   describe 'relationships' do
     it { should belong_to :supermarket }
+    it { should have_many :customers }
```

Comment:
Same thing as with your `customer_spec.rb`. We will want to set up our many-to-many associations:
```ruby
it { should have_many(:customer_items) }
it { should have_many(:customers).through(:customer_items) }
```